src/main/java/io/kestra/plugin/dbt/cli/AbstractDbt.java

```diff
@@ -4,27 +4,40 @@
 import io.kestra.core.models.annotations.PluginProperty;
 import io.kestra.core.models.tasks.RunnableTask;
 import io.kestra.core.runners.RunContext;
-import io.kestra.core.tasks.scripts.AbstractBash;
-import io.kestra.core.tasks.scripts.AbstractLogThread;
-import io.kestra.core.tasks.scripts.ScriptOutput;
+import io.kestra.core.tasks.scripts.BashService;
 import io.kestra.plugin.dbt.ResultParser;
+import io.kestra.plugin.scripts.exec.AbstractExecScript;
+import io.kestra.plugin.scripts.exec.scripts.models.DockerOptions;
+import io.kestra.plugin.scripts.exec.scripts.models.RunnerType;
+import io.kestra.plugin.scripts.exec.scripts.models.ScriptOutput;
+import io.kestra.plugin.scripts.exec.scripts.runners.AbstractLogConsumer;
+import io.kestra.plugin.scripts.exec.scripts.runners.CommandsWrapper;
+import io.kestra.plugin.scripts.exec.scripts.services.ScriptService;
 import io.swagger.v3.oas.annotations.media.Schema;
 import lombok.*;
 import lombok.experimental.SuperBuilder;
-import org.slf4j.Logger;
+import org.apache.commons.io.FileUtils;
 
+import java.io.File;
 import java.io.IOException;
 import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.ArrayList;
-
-import static io.kestra.core.utils.Rethrow.throwSupplier;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
 
 @SuperBuilder
 @ToString
 @EqualsAndHashCode
 @Getter
 @NoArgsConstructor
-public abstract class AbstractDbt extends AbstractBash implements RunnableTask<ScriptOutput> {
+public abstract class AbstractDbt extends AbstractExecScript implements RunnableTask<ScriptOutput> {
```

Comment:
I don't think the `extends AbstractExecScript` is needed, and even more it's exposed some props that would complexify the things (`interpreter`, `warningOnStdErr`)